pyproject.toml

```diff
@@ -17,7 +17,7 @@ line_length = 120
 
 [tool.black]
 line-length = 120
-target_version = ['py27', 'py35', 'py36', 'py37', 'py38']
+target_version = ['py35', 'py36', 'py37', 'py38', 'py311']
```

Comment:
why did we remove py27?

Reply:
Note: we're running our formatting pre-check using `black==21.4b4` which doesn't support python 3.10 or python 3.11. I'll revert the changes instead until we upgrade our black version.